web/pdf_find_bar.js

```diff
@@ -44,182 +45,175 @@ var PDFFindBar = (function PDFFindBarClosure() {
     }
 
     // Add event listeners to the DOM elements.
-    var self = this;
-    this.toggleButton.addEventListener('click', function() {
-      self.toggle();
+    this.toggleButton.addEventListener('click', () => {
+      this.toggle();
     });
 
-    this.findField.addEventListener('input', function() {
-      self.dispatchEvent('');
+    this.findField.addEventListener('input', () => {
+      this.dispatchEvent('');
     });
 
-    this.bar.addEventListener('keydown', function(evt) {
-      switch (evt.keyCode) {
+    this.bar.addEventListener('keydown', (e) => {
+      switch (e.keyCode) {
         case 13: // Enter
-          if (evt.target === self.findField) {
-            self.dispatchEvent('again', evt.shiftKey);
+          if (e.target === this.findField) {
+            this.dispatchEvent('again', e.shiftKey);
           }
           break;
         case 27: // Escape
-          self.close();
+          this.close();
           break;
       }
     });
 
-    this.findPreviousButton.addEventListener('click', function() {
-      self.dispatchEvent('again', true);
+    this.findPreviousButton.addEventListener('click', () => {
+      this.dispatchEvent('again', true);
     });
 
-    this.findNextButton.addEventListener('click', function() {
-      self.dispatchEvent('again', false);
+    this.findNextButton.addEventListener('click', () => {
+      this.dispatchEvent('again', false);
     });
 
-    this.highlightAll.addEventListener('click', function() {
-      self.dispatchEvent('highlightallchange');
+    this.highlightAll.addEventListener('click', () => {
+      this.dispatchEvent('highlightallchange');
     });
 
-    this.caseSensitive.addEventListener('click', function() {
-      self.dispatchEvent('casesensitivitychange');
+    this.caseSensitive.addEventListener('click', () => {
+      this.dispatchEvent('casesensitivitychange');
     });
 
     this.eventBus.on('resize', this._adjustWidth.bind(this));
   }
 
-  PDFFindBar.prototype = {
-    reset: function PDFFindBar_reset() {
-      this.updateUIState();
-    },
-
-    dispatchEvent: function PDFFindBar_dispatchEvent(type, findPrev) {
-      this.eventBus.dispatch('find', {
-        source: this,
-        type: type,
-        query: this.findField.value,
-        caseSensitive: this.caseSensitive.checked,
-        phraseSearch: true,
-        highlightAll: this.highlightAll.checked,
-        findPrevious: findPrev
-      });
-    },
-
-    updateUIState:
-        function PDFFindBar_updateUIState(state, previous, matchCount) {
-      var notFound = false;
-      var findMsg = '';
-      var status = '';
-
-      switch (state) {
-        case FindStates.FIND_FOUND:
-          break;
+  reset() {
+    this.updateUIState();
+  }
 
-        case FindStates.FIND_PENDING:
-          status = 'pending';
-          break;
+  dispatchEvent(type, findPrev) {
+    this.eventBus.dispatch('find', {
+      source: this,
+      type: type,
```

Comment:
Nit: Since you're touching this line anyway, please note that it can be simplified to just `type,`

*Please note:* The [`object-shorthand`](http://eslint.org/docs/rules/object-shorthand) ESLint rule would catch this, but given the vast number of failures it's not really feasible to enable it right now.
However, if we try to convert cases like these when we touch the code anyway, it might be slightly more doable later on (for reference, [see also this MDN article](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)).